Mage.Sets/src/mage/cards/a/AjaniValiantProtector.java

```diff
@@ -44,7 +44,7 @@ public AjaniValiantProtector(UUID ownerId, CardSetInfo setInfo) {
 
         // -11: Put X +1/+1 counters on target creature, where X is your life total. That creature gains trample until end of turn.
         Effect effect = new AddCountersTargetEffect(CounterType.P1P1.createInstance(), ControllerLifeCount.instance);
-        effect.setText("Put X +1/+1 counters on target creature, where X is your life total.");
+        effect.setText("Put X +1/+1 counters on target creature, where X is your life total"); //removed final period
```

Comment:
Please, remove that style of comments from PR (about removed symbols).

Reply:
I removed comments about removed symbols. 

I left comments about how code functions in regards to automatic capitalization and punctuation. These are in the effects files. Just let me know if you would like me to remove those as well. 

Thanks!